hadoop-mapreduce/src/main/java/org/apache/accumulo/hadoop/mapred/AccumuloFileOutputFormat.java

```diff
@@ -100,22 +81,8 @@ public void write(Key key, Value value) throws IOException {
   /**
    * Sets all the information required for this map reduce job.
    */
-  public static void setInfo(JobConf job, FileOutputInfo info) {
-    setOutputPath(job, info.getOutputPath());
-    if (info.getCompressionType().isPresent())
-      setCompressionType(job, info.getCompressionType().get());
-    if (info.getDataBlockSize().isPresent())
-      setDataBlockSize(job, info.getDataBlockSize().get());
-    if (info.getFileBlockSize().isPresent())
-      setFileBlockSize(job, info.getFileBlockSize().get());
-    if (info.getIndexBlockSize().isPresent())
-      setIndexBlockSize(job, info.getIndexBlockSize().get());
-    if (info.getReplication().isPresent())
-      setReplication(job, info.getReplication().get());
-    if (info.getSampler().isPresent())
-      setSampler(job, info.getSampler().get());
-    if (info.getSummarizers().size() > 0)
-      setSummarizers(job, info.getSummarizers().toArray(new SummarizerConfiguration[0]));
+  public static FileOutputFormatBuilder.PathParams configure() {
```

Comment:
I was thinking it would be nice to limit the cofiguration process to only JobConf for mapred and only Job for mapreduce using generics.  However I was not sure if it would work, so I tried it.  It seems to have worked out ok.   I made created milleruntime/accumulo#13